src/components/v2/V2Project/V2FundingCycleSection/NoFundingCycle.tsx

```diff
@@ -0,0 +1,27 @@
+import { Trans } from '@lingui/macro'
+import { Space } from 'antd'
+import { CardSection } from 'components/shared/CardSection'
+import SectionHeader from 'components/shared/SectionHeader'
+
+export default function NoFundingCycle() {
```

Comment:
When this PR merges, projects on the old V2 will have "No active funding cycle" shown. Once they re-launch their FC, it's back to normal